supplementary/alica_ros1/alica_tests/src/test/test_success_spam.cpp

```diff
@@ -1,49 +1,36 @@
-#include "test_alica.h"
-
-#include <alica_tests/BehaviourCreator.h>
-#include <alica_tests/ConditionCreator.h>
-#include <alica_tests/ConstraintCreator.h>
-#include <alica_tests/PlanCreator.h>
-#include <alica_tests/UtilityFunctionCreator.h>
-
+#include <alica/test/TestContext.h>
 #include <alica/test/Util.h>
-#include <engine/AlicaClock.h>
-#include <engine/AlicaEngine.h>
-#include <engine/Assignment.h>
-#include <engine/BasicBehaviour.h>
-#include <engine/DefaultUtilityFunction.h>
-#include <engine/IAlicaCommunication.h>
-#include <engine/PlanBase.h>
-#include <engine/PlanRepository.h>
-#include <engine/TeamObserver.h>
-#include <engine/model/Behaviour.h>
-#include <engine/model/Plan.h>
-#include <engine/model/RuntimeCondition.h>
-#include <engine/model/State.h>
-
+#include <alica_tests/Behaviour/Attack.h>
+#include <alica_tests/SimpleTestPlan1412252439925.h>
+#include <alica_tests/TestFixture.h>
 #include <gtest/gtest.h>
 
-namespace alica
-{
-namespace
-{
-
-class AlicaSpamSuccess : public AlicaTestFixture
+namespace alica::test
 {
-protected:
-    const char* getRoleSetName() const override { return "Roleset"; }
-    const char* getMasterPlanName() const override { return "BehaviorSuccessSpamMaster"; }
-};
 
 /**
  * Tests whether it is possible to run a behaviour in a primitive plan.
  */
-TEST_F(AlicaSpamSuccess, runBehaviour)
+TEST_F(TestFixture, runBehaviour)
 {
-    ASSERT_NO_SIGNAL
-    ae->start();
-    STEP_UNTIL(alica::test::Util::isPlanActive(ae, 1522377375148));
-    EXPECT_TRUE(alica::test::Util::isPlanActive(ae, 1522377375148));
+    // Transition to the plan corresponding to this test case
+    ASSERT_TRUE(_tc->setTransitionCond("TestMasterPlan", "ChooseTestState", "BehaviorSuccessSpamMasterState")) << _tc->getLastFailure();
```

Comment:
I don't see what this test case is testing. 

According to the name & comment it needs to check if a behaviour can run (which is in itself vague). However it does not do this because there is no check to see if the behaviour's run method is called & the plan's state machine is way too complex for this simple check. 

Another weird thing is that this test is placed in a file named `test_success_spam.cpp` which is misleading because there is no test case in this file which tests success. And the fact that there is another file called `test_success.cpp` makes this even more worse.

The idea of rewriting/improving tests is precisely to fix such inconsistencies & confusing tests